kuma/static/styles/includes/_mixins.scss

```diff
@@ -510,7 +510,8 @@ These are not dynamic but serve as mixins
 
     article > &:first-child,
     article > div:first-child:empty + &,
-    article > p:first-child:empty + & {
+    article > p:first-child:empty + &,
+    article > p:first-child:empty + p:empty + & {
```

Comment:
This fixes the&nbsp;issue when the&nbsp;first element is&nbsp;a&nbsp;`<p>` tag containing only a&nbsp;sidebar macro immediately followed by&nbsp;a&nbsp;`<blockquote>` (and previously a&nbsp;`.summary`).

Reply:
Do you have a MDN page with this issue?

Reply:
There used to&nbsp;be&nbsp;a&nbsp;few that I&nbsp;edited, but with #5193 removing the&nbsp;`.summary` special styling, finding an&nbsp;example of&nbsp;this will be&nbsp;difficult.